### DISCUSSION TITLE
How to correctly style components when using TailwindCSS

### DISCUSSION THREAD

**Discussion by wsw70** [primefaces/community#1164] [PrimeVue]
I am trying to combine PrimeVie and TailwindCSS in one, following the [documentation](https://tailwind.primevue.org/overview).

The problem I am running into is that the final effect are very low contrast windows (which is maybe a choice) but also some effects I cannot understand. One of them is the transparency of the elements, such as this `Dropdown`:

![image](https://github.com/primefaces/community/assets/1345886/73fa6095-682a-4a00-8456-7d3235b06e55)

Am I supposed to set the background myself (I though that this would be the role of a preset such as Lara)? If so - how?

**Comment by tugcekucukoglu** [comment]
I cannot replicate the issue: https://tailwind.primevue.org/dropdown/

Could you share your preset with a reproducer demo?
